api/cpp/lib.rs

```diff
@@ -230,3 +230,7 @@ pub unsafe extern "C" fn slint_set_xdg_app_id(_app_id: &SharedString) {
 pub unsafe extern "C" fn slint_detect_operating_system() -> OperatingSystemType {
     i_slint_core::detect_operating_system()
 }
+
+pub unsafe extern "C" fn open_url(url: &SharedString) {
```

Comment:
This needs `#[unsafe(no_mangle)]`
(And the function doesn't need to be marked unsafe since it is not taking a pionter)